### PR TITLE
(MODULES-9596) - add changelog generator

### PR DESCRIPTION
**Pull request by lionce**
Convert windows modules to use changelog generator

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Changelog
-All notable changes to this project will be documented in this file.
+# Change log
```
Comment (michaeltlombardi) [inline CHANGELOG.md:1]: Is changeling being split into two words intentional? Maybe I just haven't noticed this before 😭😭😭

**Review by michaeltlombardi — APPROVED** [review]
One minor question but nothing to hold up a merge for long.

File: CHANGELOG.md (hunk 1 above)
Comment (RandomNoun7) [reply CHANGELOG.md:1]: LOL. Same exact thought.

**Review by RandomNoun7 — APPROVED** [review]

File: Rakefile [hunk 2]
```diff
@@ -1,8 +1,82 @@
...
+end
+
+PuppetLint.configuration.send('disable_relative')
+
+if Bundler.rubygems.find_name('github_changelog_generator').any?
+  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+    raise "Set CHANGELOG_GITHUB_TOKEN environment variable eg 'export CHANGELOG_GITHUB_TOKEN=valid_token_here'" if Rake.application.top_level_tasks.include? "changelog" and ENV['CHANGELOG_GITHUB_TOKEN'].nil?
+    config.user = "#{changelog_user}"
+    config.project = "#{changelog_project}"
+puts "-------#{changelog_project}=#{changelog_project}"
```
Comment (ThoughtCrhyme) [inline Rakefile:42]: This is causing Jenkins CI to fail a la https://jenkins-master-prod-1.delivery.puppetlabs.net/view/modules/view/windows/view/iis/view/master/job/forge-windows_puppetlabs-iis_init-merge_master/245/console

**Pull request merged**